lib/crypto.js

```diff
@@ -100,7 +100,8 @@ Hash.prototype.update = function update(data, encoding) {
 
 Hash.prototype.digest = function digest(outputEncoding) {
   outputEncoding = outputEncoding || exports.DEFAULT_ENCODING;
-  return this._handle.digest(outputEncoding);
+  // Explicit conversion for backward compatibility
+  return this._handle.digest(String(outputEncoding));
```

Comment:
Use <code>&#96;${outputEncoding}&#96;</code>; otherwise symbols (which were not allowed) will be converted to string also.